public/assets/harvard.js

```diff
@@ -201,3 +201,32 @@ function responsive_search(){
     }
   });
 }
+
+// Keyboard controls for bootstrap dropdown menus
+const toggleDropdown = (event, currentElement) => {
+  if (event.keyCode === 13) {
+      event.preventDefault();
+      currentElement.dropdown("toggle");
+      if (currentElement.parent().hasClass('open')) {
+          const listItems =  currentElement.next("ul").find("a");
+          listItems[0].focus();
+      }
+  } else if (event.keyCode === 27) {
+      currentElement.dropdown("toggle");
+  }
+}
+
+$(window).on('load', function() {
+  $(".dropdown-toggle").bind("keydown", function(event) {
```

Comment:
`.bind` has been deprecated as of jQuery 3.0, and while I think we're not close, I don't see any reason not to use `.on` here.  Also, if it doesn't have a compelling reason to be directly bound, it's good to get in the habit of using delegated handlers, for performance reasons:

```javascript
$(window).on('keydown', '.dropdown-toggle', toggleDropdown);
```
(assuming you change toggleDropdown as specified above)